adsft/extraction.py

```diff
@@ -414,38 +409,6 @@ def open_xml(self):
 
         return raw_xml
 
-    def _remove_keeping_tail(self, element):

```

Comment:
Probably you already check it, but I'm afraid that not using this function will make us lose text as explained in the method comment. Do we have a unit test for this? If not, could we create one?

Reply:
Here's the definition:
https://lxml.de/api/lxml.html-pysrc.html#HtmlMixin.drop_tree

where is states:
Removes this element from the tree, including its children and 
text. The tail text is joined to the previous element or parent. 

Should I add a unit test if it is a function defined outside of this program?

Reply:
Yes, let’s add a unit test for this 👍

Reply:
unit test added, see test_that_the_tail_is_preserved in test_extraction.py